arelle/plugin/validate/DBA/ValidationPluginExtension.py

```diff
@@ -9,6 +9,8 @@
 from arelle.utils.validate.ValidationPlugin import ValidationPlugin
 from .PluginValidationDataExtension import PluginValidationDataExtension
 
+import re
```

Comment:
We use the more performant regex library throughout Arelle.

```suggestion
import regex as re
```